cardea-server/src/test/java/ca/on/oicr/gsi/cardea/server/MockCase.java

```diff
@@ -126,7 +126,6 @@ private static Case makeCase(String donorName, String assayName, String projectN
     Assay assay = mock(Assay.class);
     when(assay.getName()).thenReturn(assayName);
     when(assay.getDescription()).thenReturn(assayName);
-    when(kase.getAssay()).thenReturn(assay);
```

Comment:
The mock assay is no longer used, so should be removed. `kase.getAssayName` and `kase.getAssayDescription` should be mocked to maintain previous behaviour